sw/otbn/crypto/rsa_verify_3072.s

```diff
@@ -335,6 +335,101 @@ montmul:
 
   ret
 
+/**
+ * Variable time 3072-bit modular exponentiation with exponent 3
+ *
+ * Returns: C = modexp(A,3) = mod M
+ *
+ * The squared Montgomery modulus RR and the Montgomery constant m0' have to
+ * be provided at the appropriate locations in dmem.
+ *
+ * Flags: Flags have no meaning beyond the scope of this subroutine.
+ *
+ * The base bignum A is expected in the input buffer, the result C is written
+ * to the output buffer. Note, that the content of the input buffer is
+ * modified during execution.
+ *
+ * @param[in]  dmem[m0inv] pointer to m0' in dmem
+ * @param[in]  dmem[rr] pointer to RR in dmem
+ * @param[in]  dmem[in_mod] pointer to first limb of modulus M in dmem
+ * @param[in]  dmem[in_buf] pointer to buffer with base bignum
+ * @param[in]  dmem[out_buf] pointer to output buffer
+ *
+ * clobbered registers: x2, x5 to x13, x16 to x21, x29
```

Comment:
I wonder if we should invent a convention for clobbers.

https://stackoverflow.com/questions/48833748/does-doxygen-have-a-clobber-like-parameter-specification

Reply:
If we make something machine-readable, the information-flow stuff in #9627 can check it (although you'd have to parse it out of the comment first).